### ISSUE TITLE
Re: geocaching app

### ISSUE THREAD

**Issue by evilbunny2008** [Speiburger/geocaching#3]
I am usually pretty responsive about bugs and what not, and I feel you should have at least filed a bug report or contacted me by some means.

Going forward it seems a little silly not to combine efforts rather than having the app twice in the app store.

**Comment by Speiburger** [comment]
There were already two bug reports about the exact problem and also two pull requests addressing those problems by  vpzomtrrfrt. Another user also wanted to take over the app. Since those pull requests and issues remained open since May 4 2019 with no response form you, I figured that you abandoned the project. I did not mean to offend you in any way and I just wanted to help the people that need a geocaching app for UT. 

I definitely think that we should try to combine our efforts, you are right about that. I look forward to working with you in the future if you are interested.

**Comment by evilbunny2008** [comment]
I spent a bit trying to figure out why I never got emails, and for what ever reason my account wasn't set to notify me.

**Comment by Speiburger** [comment]
Oh, that explains a lot! I now created a pull request with my changes and a (hopefully) complete list of all of them. After your open store app gets an Update, I will take my app down, this is just so there isn't a working app for some time. I hope I could help at least with my additions and will definitely help with further development. My solutions right now are sometimes not the most efficient or worse, because I wanted to get a working app out ASAP. I will address those problems in further pull requests tho.

**Comment by evilbunny2008** [comment]
Apart from adding a found column in geocaches were there any other database changes?

I'm patching my code to add it rather than expect people reinstall the app for new fields.

**Comment by Speiburger** [comment]
I don't think so. I added just the found category in all database functions

**Comment by evilbunny2008** [comment]
I think most stuff merged ok, however I'm stuck on this line:

Image { id: image; source: getMapMarker(cachetype, found); width: units.gu(5); height: units.gu(5) }

and get this error:

file:///opt/click.ubuntu.com/geocaching.evilbunny/0.0.9/qml/MapScreen.qml:163: ReferenceError: found is not defined

Yet if I refresh a cache the icons shows up fine.

**Comment by Speiburger** [comment]
Small oversight on my part. I hope it works now, if there are any other problems that I missed, please contact me!

**Comment by evilbunny2008** [comment]
This still doesn't fix the issue, I think all the icons missing are actually found caches and refreshing a cache seems to show all icons and does not check found

**Comment by evilbunny2008** [comment]
I fixed my problem, when I altered the table I assumed it would put 0 in the field, but it set them to null. 

The next problem is fixing the refresh, I'm not sure what it's doing but it doesn't seem to work or work properly.

**Comment by evilbunny2008** [comment]
Pretty sure all the refresh and found issues have been sorted, your feedback would be welcome.

**Comment by Speiburger** [comment]
I tested the app again - no problems found!
Weird thing is that I could not replicate any problems you had - even when I first cloned your repo after you merged my pull request.
I assume this is because of the small regional differences, like date or time format. 

Anyways, I am going to try out the app today in practical use during a quick tour, if I find any problems there I will message you. Otherwise I think the app is ready for an update in the Open Store!
